src/apply.jl

```diff
@@ -101,3 +101,27 @@ function apply!(table::T, t::Transform; cols=_get_cols(table), kwargs...)::T whe
 
     return table
 end
+
+"""
+    apply_append(A::AbstractArray, ::Transform; append_dim, kwargs...)
+
+Applies the [`Transform`](@ref) to `A` and returns the result in a new array where the output
+is appended to `A` along the `append_dim` dimension. The remaining `kwargs` correspond to
+the usual [`Transform`](@ref) being invoked.
+"""
+function apply_append(A::AbstractArray, t; append_dim, kwargs...)::AbstractArray
```

Comment:
Looking at some tests repeating `dims=x, append_dim=x` made me think, would it make sense to set the default value for `append_dim` as `dims` inside the function? Maybe that only works when `dims` is single, as is currently the case.

Reply:
It definitely does for linear combination... I'll have a look and see if it makes sense in general and make the change if so.

Reply:
I think I'll leave this for another PR - I have an idea or two of how to make this work but it'll require another review

Reply:
https://github.com/invenia/FeatureTransforms.jl/issues/70